content/courses/Ada_For_The_C_Embedded_Developer/chapters/06_Translation.rst

```diff
@@ -762,19 +762,53 @@ This technique is called overlays for serialization. Now, any operation that we
 perform on :ada:`B` will have a direct impact on :ada:`V`, since both are using
 the same memory location.
 
+The approach that we use in this section relies on the :ada:`Address`
+attribute. Another approach would be to use unchecked conversions, which we'll
+discuss in the :ref:`next section <OverlaysVsUncheckedConversions>`.
+
+We should add the :ada:`Volatile` aspect to the declaration to cover the case
+when both objects can still be changed independently |mdash| they need to be
+volatile, otherwise one change might be missed. This is the updated
+declaration:
+
+[Ada]
+
+.. code-block:: ada
+
+    B : Bit_Field (0 .. V'Size - 1) with Address => V'Address, Volatile;
+
+Using the :ada:`Volatile` aspect is important at high level of optimizations.
+You can find further details about this aspect in the section about the
+:ref:`Volatile and Atomic attributes <VolatileAtomicData>`.
+
+Another important aspect is :ada:`Import`, which must be set to :ada:`True`.
+When used in the context of object declarations, it'll avoid default
+initialization which could overwrite the existing content while creating the
+overlay |mdash| see an example in the admonition below. The declaration now
+becomes:
+
+.. code-block:: ada
+
+    B : Bit_Field (0 .. V'Size - 1)
+      with
+        Address => V'Address,
+        Import  => True,
```

Comment:
Style note, I don't think you need Import => True. Import by itself implies True.